### ISSUE TITLE
Quota overlay in NC 18.0.1 not working

### ISSUE THREAD

**Issue by tflidd** [nextcloud/server#19758]
### Steps to reproduce
1. Open NC 18.0.1 in webbrowser
2. Go to quota page
3. Try to change quota for one of the first users

### Expected behaviour
Proper overlay menu to change the value.

### Actual behaviour
Tell us what happens instead
![image](https://user-images.githubusercontent.com/8111200/75769336-48e0b000-5d46-11ea-93e3-111e35c46f3a.png)

The stuff from underneath is on top and the items can't be seen properly

### Server configuration

**Operating system:** FreeBSD

**Web server:** Nginx

**Nextcloud version:** 18.0.1

**Updated from an older Nextcloud/ownCloud or fresh install:** Updated

**List of activated apps:**
<details>
<summary>

</summary>

```
If you have access to your command line run e.g.:
  - accessibility: 1.4.0
  - calendar: 2.0.2
  - cloud_federation_api: 1.1.0
  - comments: 1.8.0
  - dav: 1.14.0
  - federatedfilesharing: 1.8.0
  - federation: 1.8.0
  - files: 1.13.1
  - files_automatedtagging: 1.8.0
  - files_external: 1.9.0
  - files_pdfviewer: 1.7.0
  - files_rightclick: 0.15.2
  - files_sharing: 1.10.1
  - files_trashbin: 1.8.0
  - files_versions: 1.11.0
  - firstrunwizard: 2.7.0
  - logreader: 2.3.0
  - lookup_server_connector: 1.6.0
  - nextcloud_announcements: 1.7.0
  - oauth2: 1.6.0
  - password_policy: 1.8.0
  - photos: 1.0.0
  - privacy: 1.2.0
  - provisioning_api: 1.8.0
  - recommendations: 0.6.0
  - serverinfo: 1.8.0
  - settings: 1.0.0
  - sharebymail: 1.8.0
  - systemtags: 1.8.0
  - text: 2.0.0
  - twofactor_backupcodes: 1.7.0
  - twofactor_totp: 4.1.2
  - twofactor_u2f: 5.0.2
  - updatenotification: 1.8.0
  - user_external: 0.8.0
  - viewer: 1.2.0
  - workflowengine: 2.0.0

```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "***REMOVED SENSITIVE VALUE***"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
        "htaccess.RewriteBase": "\/",
        "dbtype": "mysql",
        "version": "18.0.1.3",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "user_backends": [
            {
                "class": "OC_User_IMAP",
                "arguments": [
                    "{***REMOVED SENSITIVE VALUE***:993\/imap\/ssl}"
                ]
            }
        ],
        "updater.release.channel": "beta",
        "loglevel": 0,
        "log_rotate_size": 10485760,
        "maintenance": false,
        "allow_user_to_change_display_name": true,
        "theme": "",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0
        },
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "sendmail",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpdebug": true,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***"
    }
}

```
</details>

### Client configuration
**Browser:** FF 73.0.1 / Chrome 80.0.3987.122

**Operating system:** Win 10

In FF, it shows a lot of non-loaded *.js resources because of CSP policy. CSP header:
```
default-src 'none';
base-uri 'none';
manifest-src 'self';
script-src 'self';
style-src 'self' 'unsafe-inline';
img-src 'self' data: blob:;
font-src 'self' data:;
connect-src 'self';
media-src 'self';
frame-ancestors 'self';
form-action 'self'
```
Chrome doesn't show these errors and seems to load all the *.js files.

**Comment by hobesman** [comment]
Just fyi, the same happens for the drop-down for groups on the same page.

Oddly, if the user being managed is at the bottom of the screen, the drop down goes up instead of down. When it goes up, the list is properly displayed on top and is clickable.

**Comment by GretaD** [comment]
This is fixed on 18.04+, can you please recheck?

**Comment by szaimen** [comment]
yes, seems to be fixed in the meantime
![image](https://user-images.githubusercontent.com/42591237/120473779-ad564900-c3a7-11eb-8c74-c8a849a3fa9a.png)
